web_src/js/features/repo-issue-edit.ts

```diff
@@ -78,10 +78,21 @@ async function onEditContent(event) {
 
   comboMarkdownEditor = getComboMarkdownEditor(editContentZone.querySelector('.combo-markdown-editor'));
   if (!comboMarkdownEditor) {
+    const opts = {};
+    opts.onContentChanged = (editor) => {
+      const saveButton = editContentZone.querySelector('.ui.primary.button');
+      const isUploading = editor.isUploading();
+      if (saveButton) {
```

Comment:
The `saveButton` should always be there. So is the `if` really necessary?

Reply:
In `repo-issue.ts`, there are null check logic for statusButton and commentButton. for consistency i make null check logic on here.